pkg/networkmanager/ip-settings.jsx

```diff
@@ -182,45 +207,45 @@ export const IpSettingsDialog = ({ topic, connection, dev, settings }) => {
-                                    addresses.map((item, index) =>
-                                        i === index
-                                            ? addressIpv4Helper(value)
-                                            : item
-                                    ))} />
-                            </FormGroup>
-                            <FormGroup fieldId={idPrefix + "-netmask-" + i} label={prefixText} className="pf-m-4-col-on-sm">
-                                <TextInput id={idPrefix + "-netmask-" + i} value={address.netmask} onChange={(_event, value) => setAddresses(
-                                    addresses.map((item, index) =>
-                                        i === index
-                                            ? { ...item, netmask: value }
-                                            : item
-                                    ))} />
-                            </FormGroup>
-                            <FormGroup fieldId={idPrefix + "-gateway-" + i} label={_("Gateway")} className="pf-m-4-col-on-sm">
-                                <TextInput id={idPrefix + "-gateway-" + i} value={address.gateway} onChange={(_event, value) => setAddresses(
-                                    addresses.map((item, index) =>
-                                        i === index
-                                            ? { ...item, gateway: value }
-                                            : item
-                                    ))} />
-                            </FormGroup>
-                            <FormGroup className="pf-m-1-col-on-sm remove-button-group">
-                                <Button variant='plain'
-                                        isDisabled={method == 'manual' && i == 0}
-                                        onClick={() => setAddresses(addresses.filter((_, index) => index !== i))}
-                                        aria-label={_("Remove item")}
-                                        icon={<TrashIcon />} />
-                            </FormGroup>
-                        </Grid>
-                    );
-                })}
+                <Grid hasGutter>
+                    {addresses.map((address, i) => {
+                        return (
+                            <React.Fragment key={i}>
+                                <FormGroup fieldId={idPrefix + "-address-" + i} label={_("Address")} className="pf-m-6-col-on-sm">
+                                    <TextInput id={idPrefix + "-address-" + i} value={address.address} onChange={(_event, value) => setAddresses(
+                                        addresses.map((item, index) =>
+                                            i === index
+                                                ? addressIpv4Helper(value, i)
+                                                : item
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22330-e10cfba6-20250915-113603-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)